rondb/src/main/java/site/ycsb/db/clusterj/table/UserTableHelper.java

```diff
@@ -59,8 +58,7 @@ public static DynamicObject createDTO(ClassGenerator classGenerator, Session ses
     return persistable;
   }
 
-  private static void setFieldValue(DynamicObject persistable, String colName, byte[] value,
-                                    int lenght) {
+  private static void setFieldValue(DynamicObject persistable, String colName, byte[] value, int lenght) {
```

Comment:
why would you do such a change?

Reply:
Autoformatter.

Reply:
Please revert such changes. The existing code is correctly formatted in this case

Reply:
I just ran Checkstyle over it and apparently I didn't break anything. I'll continue running an auto formatter over the code that I write and in this case I removed an unnecessary newline. I personally found it easier to read this way. But this is a PR towards your repo anyways, so feel free to change anything back before it's merged to logicalclocks, where I'll rebase from again afterwards.